test/index.test.ts

```diff
@@ -229,6 +229,23 @@ describe('FiatConnect SDK', () => {
       expect(getHeadersMock).not.toHaveBeenCalled()
     })
   })
+  describe('isLoggedIn', () => {
+    it('returns false when sessionExpiry does not exist', () => {
+      expect(client.isLoggedIn()).toBeFalsy()
+    })
+    it('returns false when sessionExpiry exists but is too old', () => {
+      client._sessionExpiry = new Date('2022-05-01T00:00:00+0000')
+      const mockNow = new Date('2022-05-02T00:00:00+0000').getTime()
+      jest.spyOn(global.Date, 'now').mockReturnValueOnce(mockNow)
+      expect(client.isLoggedIn()).toBeFalsy()
+    })
+    it('returns true when sessionExpiry exists and is not too old', () => {
+      client._sessionExpiry = new Date('2022-05-03T00:00:00+0000')
+      const mockNow = new Date('2022-05-02T00:00:00+0000').getTime()
+      jest.spyOn(global.Date, 'now').mockReturnValueOnce(mockNow)
+      expect(client.isLoggedIn()).toBeTruthy()
+    })
+  })
   describe('_ensureLogin', () => {
     const mockLogin = jest.spyOn(client, 'login')
     it('calls login and returns successfully if sessionExpiry is not set', async () => {
```

Comment:
nit: can remove one of the first two tests here since this is covered by `isLoggedIn`